galore/cli/galore.py

```diff
@@ -305,6 +305,24 @@ def get_parser():
         '--overlay_label', type=str, default=None,
         help='Legend label for data overlay'
         )
+    parser.add_argument(
+        "--no-total",
+        action="store_false",
+        dest="total",
+        help="Don't plot the total photoelectron spectrum (only works with --pdos flag)",
+    )
+    parser.add_argument(
+        '--legend-cutoff', type=float, default=2, dest="legend_cutoff",
+        help='Cutoff in %% of max y-value in the plot range that determines if a line is given a '
+             'legend label (default: 2)')
+    parser.add_argument(
+        "--no-fill",
+        action="store_false",
+        dest="fill",
+        help="Don't colour-fill the photoelectron spectrum plot",
+    )
```

Comment:
I'd propose that the outputs look nicer like this, similar to the default in `sumo` DOS plotting for example, and I don't think it breaks any previous arguments? So worth keeping as default and making a new release when this is merged?

But ofc happy with whatever you choose @ajjackson!

Reply:
Right, making a new release is an option! My point was that if behaviour changes given the same arguments as before this would usually be associated with a *major* release version. (i.e. 1.0)

There isn't really a problem with doing that, either. But if going to 1.0 we should have a think about any other "breaking" changes that we would like to make at the same time. Dropping some old Python versions, say!

Reply:
Aha! Well, they are pretty ancient so maybe we can deprecate these versions (and adopt some newer Python features!) We just need to make sure that the setup.py etc. is also updated correctly so that Pip does not install an incompatible version for anyone.